apps/www/app/docs/sidebar.tsx

```diff
@@ -21,6 +23,7 @@ import {
 import { usePathname } from "next/navigation"
 import { useEffect, useRef, useState } from "react"
 import { AiOutlineClose, AiOutlineMenu } from "react-icons/ai"
+import { ChevronRightIcon } from "../../../../packages/react/src/components/icons"
```

Comment:
Can we use icons from `react-icons/ai` here

Reply:
done